src/components/PageMetaData/PageMetaData.tsx

```diff
@@ -1,17 +1,19 @@
 import { Helmet } from 'react-helmet'
 
+import { useEnvironment } from '../../hooks/useEnvironment'
+
 interface PageMetaDataProps {
-  title: string
-  description: string
-  noIndex: boolean
+  title?: string
+  description?: string
 }
 
-export function PageMetaData({ title: pageTitle, description, noIndex }: PageMetaDataProps) {
+export function PageMetaData({ title = 'Swapr', description }: PageMetaDataProps) {
+  const { isProduction } = useEnvironment()
   return (
     <Helmet>
-      <meta name="title" content={pageTitle} />
-      <meta name="description" content={description} />
-      {noIndex && <meta name="robots" content="noindex,nofollow" />}
+      <title>{title}</title>
+      {description && <meta name="description" content={description} />}
+      {!isProduction && <meta name="robots" content="noindex,nofollow" />}
```

Comment:
I wonder  if there is a chance that we want to use noindex on production, such as on the login page

Reply:
I dont see a roadmap where swapr will have a login screen. Also I didn't see any web3 apps with login since all are wallet connect modals.

Reply:
Yeah, I think will continue without login pages, only connect wallet modal.